easybuild/easyconfigs/h/h5py/h5py-2.9.0-fosscuda-2019b.eb

```diff
@@ -0,0 +1,32 @@
+easyblock = 'PythonPackage'
+
+name = 'h5py'
+version = '2.9.0'
```

Comment:
@Flamefire You should add a `versionsuffix` here since you're depending on a specific `Python` version (via `SciPy-bundle`):

```python
versionsuffix = '-Python-%(pyver)s'
```

and:

```python
dependencies = [
    ('Python', '3.7.4'),
    ('SciPy-bundle', '2019.10', versionsuffix),
    ...
```

Note that this also implies renaming the easyconfig file.

Reply:
I see. As this is based on https://github.com/easybuilders/easybuild-easyconfigs/blob/develop/easybuild/easyconfigs/h/h5py/h5py-2.9.0-fosscuda-2019a.eb the same reasoning applies as in https://github.com/easybuilders/easybuild-easyconfigs/pull/9251#issuecomment-552592489

Maybe we discuss this in #9251 and apply the same resolution here afterwards

Reply:
Here there's little room for discussion, since the `SciPy-bundle` dependency requires a specific `Python` version (which was done to avoid getting stuck with older Python 2 compatible versions of `numpy` & co, see the discussion in #9016)...

Reply:
I see. Thanks